source/common/types/hardware-wallets.types.ts

```diff
@@ -1,7 +1,7 @@
 import { BridgeInfo, Device as TrezorDevice, UdevInfo } from 'trezor-connect';
 
 export type BIP32Path = Array<number>;
-export type LedgerModel = 'nanoS' | 'nanoX';
+export type LedgerModel = 'nanoS' | 'nanoSP' | 'nanoX';
```

Comment:
Perhaps we could call it `nanoSPlus`? I think it would make things clearer and easier to understand for people not familiar with the codebase.

Reply:
This is coming from the library, I can't change it.

https://github.com/LedgerHQ/ledgerjs/blob/23ba9fa883db9f49f81de236c9232df2c5ca1020/packages/devices/src/index.ts#L27